llvm/lib/Transforms/Instrumentation/SanitizerCoverage.cpp

```diff
@@ -1013,19 +1051,10 @@ void ModuleSanitizerCoverage::InjectCoverageAtBlock(Function &F, BasicBlock &BB,
                       ConstantInt::get(IntptrTy, Idx * 4)),
         PtrTy);
     if (Options.GatedCallbacks) {
-      if (!FunctionGateCmp) {
-        // Create this in the entry block
-        assert(IsEntryBB);
-        FunctionGateCmp = CreateFunctionLocalGateCmp(IRB);
-      }
-      // Set the branch weights in order to minimize the price paid when the
-      // gate is turned off, allowing the default enablement of this
-      // instrumentation with as little of a performance cost as possible
-      auto Weights = MDBuilder(*C).createBranchWeights(1, 100000);
```

Comment:
This can be converted to `auto Weights = MDBuilder(*C).createUnlikelyBranchWeights();`
In a separate patch if possible.

Reply:
+1

`MDNode *MDBuilder::createUnlikelyBranchWeights() {
  // Value chosen to match UR_NONTAKEN_WEIGHT, see BranchProbabilityInfo.cpp
  return createBranchWeights(1, (1U << 20) - 1);
}`